apps/meet-app/webapp/src/component/common/PreLoader.tsx

```diff
@@ -93,14 +87,14 @@ const PreLoader = (props: PreLoaderProps) => {
                 alt="logo"
                 width="150"
                 height="auto"
-                src={require("../../assets/images/wso2-logo.svg").default}
-              ></img>
+                src={require("@assets/images/wso2-logo.svg").default}
```

Comment:
Shall we import this at the top and use it here

Reply:
both ways are ok. but the suggested approach is more clean. +1 for the suggestion.